go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -478,6 +479,29 @@
         end
       end
     end
+
+    context "for a monorepo" do
+      let(:project_name) { "monorepo" }
+      let(:directory) { "cmd" }
```

Comment:
This was a bug: the directory is typically specified with a leading slash https://github.com/dependabot/dependabot-core/pull/2863